tests/functional/commands/validate/lib/test_sam_template_validator.py

```diff
@@ -157,8 +149,18 @@ class TestValidate(TestCase):
         ("tests/functional/commands/validate/lib/models/sns_topic_outside_template.yaml"),
         ("tests/functional/commands/validate/lib/models/sqs.yaml"),
         ("tests/functional/commands/validate/lib/models/streams.yaml"),
-        ("tests/functional/commands/validate/lib/models/unsupported_resources.yaml"),
+        ("tests/functional/commands/validate/lib/models/unsupported_resources.yaml")
+    ]
         (
+            "tests/functional/commands/validate/lib/models/function_with_conditional_managed_policy_and_ref_no_value.yaml"
```

Comment:
nitpick: these lines are differently formatted than others.

Reply:
Thank you for noticing. This is still a work in progress. I will update and push the changes